packages/components/atoms/error-message-atom/src/ErrorMessage.atom.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { NavLink } from '../../nav-link-atom/src';
+
+export interface ErrorMessageProps {}
```

Comment:
We can start taking the props out if we don't need them. Plop generates them just incase they are needed